aea/decision_maker/messages/transaction.py

```diff
@@ -291,6 +292,7 @@ def _is_consistent(self) -> bool:
             elif self.performative in {
                 self.Performative.PROPOSE_FOR_SIGNING,
                 self.Performative.REJECTED_SIGNING,
+                self.performative.PROPOSE_FOR_CONTRACT
```

Comment:
I think this is a propose for signing as well.

Reply:
Looks like that the PROPOSE_FOR_SIGNING is working too. The only thing that I have to figure out is how I am going to send the transaction after signing

Reply:
You can access the ledger apis on the skill context, hence this will work: `ledger_apis.apis.get(tx_message.ledger_id).send_raw_transaction(tx_signed=signature,
                                                                                  tx_type=tx_message.tx_id)`

Reply:
`tx_type=tx_message.tx_id` not sure about this part.. what does it do?